.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,137 @@
+on:
+  # Branch deletion
+  delete:
+  pull_request:
+  push:
```

Comment:
I think maybe just `pull_request` so that someone has to open an actual PR to start doing cloud stuff and can leave WIP branches without doing so?

Reply:
I think that we still need to the push for deploying when there are pushes to the master branch?